debian/changelog

```diff
@@ -1,3 +1,9 @@
+asterisk (8:16.6.2-1~4) wazo-dev-buster; urgency=medium
```

Comment:
it's not ~wazo4 instead of ~4?

Reply:
yeah it should have been `~wazo4`
I'll have to force a rebuild with the good version because ~4 is lower than ~wazo3